src/hotspot/share/utilities/globalDefinitions.hpp

```diff
@@ -297,6 +297,9 @@ inline jdouble jdouble_cast(jlong x);
 const jlong min_jlong = CONST64(0x8000000000000000);
 const jlong max_jlong = CONST64(0x7fffffffffffffff);
 
+// for timer info max values which include all bits, was CONST64(0x7fffffffffffffff);
```

Comment:
```suggestion
// for timer info max values which include all bits, was CONST64(0xffffffffffffffff);
```

Reply:
Or just remove the `was CONST64` part since people can find this in git history.